pkgs/tools/misc/bkyml/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3
+}:
+
+with python3.pkgs; buildPythonApplication rec {
```

Comment:
```suggestion
with python3.pkgs;

buildPythonApplication rec {
```

Reply:
I found that `with python3.pkgs; buildPythonApplication rec {` is [being used](https://github.com/NixOS/nixpkgs/search?q=python3.pkgs%3B+buildPythonApplication+rec+%7B) quite many times across the project this is why I used it. Can you please link to a style guide? Anyway, this is not a big deal, I am okay to change it if this will make someone happy. Done in f80ee2a please have a look.